openapi/openapi.yaml

```
@@ -409,11 +409,76 @@ components:
               description: >-
                 Pipfile stating software stack (direct dependencies) of an
                 application.
-              additionalProperties: {}
+              example: >-
+                {
+                  "source": [
+                    {
+                      "name": "pypi",
+                      "url": "https://pypi.org/simple",
+                      "verify_ssl": true
+                    }
+                  ],
+                  "packages": {
+                    "numpy": "*"
+                  },
+                  "pipenv": {
+                    "allow_prereleases": false
+                  },
+                  "requires": {
+                    "python_version": "3.8"
+                  }
+                }
             requirements_locked:
               type: object
               description: Pipfile.lock with fully pinned down and resolved software stack.
-              additionalProperties: {}
+              example: >-
```

`>-` will convert JSON into a string. You can remove `>-` and keep JSON (that gets correctly injected into the YAML file):

```suggestion
              example:
```